ios/brave-ios/Sources/Web/Tab.swift

```diff
@@ -1094,27 +876,9 @@ class TabWebView: WKWebView, MenuHelperInterface {
     if #available(iOS 16.4, *) {
       isInspectable = true
     }
-
-    updateBackgroundColor()
-    Preferences.General.nightModeEnabled.observe(from: self)
   }
 
-  private func updateBackgroundColor() {
-    if Preferences.General.nightModeEnabled.value {
-      let color = UIColor(braveSystemName: .containerBackground)
-      backgroundColor = color
-      scrollView.backgroundColor = color
-      // WKWebView flashes white screen on load regardless of background colour assignments without
-      // setting `isOpaque` to false
-      isOpaque = false
-    } else {
-      backgroundColor = nil
-      scrollView.backgroundColor = nil
-      isOpaque = true
-    }
-  }
```

Comment:
@Brandon-T  Do we still need this after your NTP bg color fix?

Reply:
Doesn't seem like it!

Reply:
Not needed